spring-boot-starter-rocketmq/README.md

```diff
@@ -0,0 +1,207 @@
+# spring-boot-starter-rocketmq
+
+[中文](./README_zh_CN.md)
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
+
+Help developers quickly integrate [RocketMQ](http://rocketmq.apache.org/) in [Spring Boot](http://projects.spring.io/spring-boot/). Support the Spring Message specification to facilitate developers to quickly switch from other MQ to RocketMQ.
+
+
+Features:
+
+- [x] synchronous transmission
+- [x] synchronous ordered transmission
+- [x] asynchronous transmission
+- [x] asynchronous ordered transmission
+- [x] orderly consume
+- [x] concurrently consume(broadcasting/clustering)
+- [x] One-way transmission
+- [ ] transaction transmission
+- [ ] Pull consume 
+
+## Quick Start
```

Comment:
Did I miss something? I couldn't run this `quick start`, maybe we need a configuration in `quick start`?


```
***************************
APPLICATION FAILED TO START
***************************

Description:

A component required a bean of type 'org.apache.rocketmq.spring.starter.core.RocketMQTemplate' that could not be found.
	- Bean method 'rocketMQTemplate' in 'RocketMQAutoConfiguration' not loaded because @ConditionalOnBean (types: org.apache.rocketmq.client.producer.DefaultMQProducer; SearchStrategy: all) did not find any beans


Action:

Consider revisiting the conditions above or defining a bean of type 'org.apache.rocketmq.spring.starter.core.RocketMQTemplate' in your configuration.
```

Reply:
I provided a complete demo， please see https://github.com/aqlu/rocketmq-demo

Reply:
Hi, could you please add this demo to README?

Reply:
It's already add. 
`Here are some key points listed, the complete example, please refer to: rocketmq-demo`

Reply:
Got it, I will try this demo later, thanks~